otwcode/otwarchive
# AO3-5391 Remove unused taggable methods

## Issue

https://otwarchive.atlassian.net/browse/AO3-5391

## Purpose

Removes unused code! :tada: (Note that it also removes something from the config file.)

ETA: One method -- `fandom_tags` -- turned out to be used in showing fandoms on external work blurbs, so I replaced it with a different way of doing that (the same way we use on work blurbs).

ETA 2: Two methods, `freeform_tags` and `warning_tags`, turned out to be used in showing tags on series blurbs when the user has tag-hiding preferences enabled. I switched them to the usual `freeforms` and `warnings` methods and updated the tests that didn't quite catch this issue.

## Testing

* Make sure external works don't 500.
* Set your preferences to hide warnings and additional tags, and then make sure you can reveal warnings and freeforms on series blurbs (that is, the blurb on /users/USERNAME/series -- not the work blurbs on the /series/1234 pages.)
